app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/deregistration/DeregistrationInfo.scala

```diff
@@ -43,8 +43,8 @@ case class DeregistrationInfo(deregReason: DeregistrationReason,
   private val validateReducedTurnover: JsResult[DeregistrationInfo] = turnoverBelowThreshold match {
     case None =>
       JsError("turnoverBelowThreshold is mandatory when deregReason is belowThreshold")
-    case Some(x) if x.belowThreshold == BelowNext12Months && x.whyTurnoverBelow.isEmpty =>
-      JsError("whyTurnoverBelow is mandatory when belowThreshold is belowNext12Months")
+//    case Some(x) if x.belowThreshold == BelowNext12Months && x.whyTurnoverBelow.isEmpty =>
```

Comment:
Why is this commented out now?

Reply:
I meant to delete that because it's  not an Option anymore.